src/app/vault/ciphers.component.html

```diff
@@ -47,7 +47,11 @@
                                     <i class="fa fa-fw fa-clone" aria-hidden="true"></i>
                                     {{'copyPassword' | i18n}}
                                 </a>
-                                <a class="dropdown-item" href="#" appStopClick *ngIf="c.login.canLaunch"
+                                <a class="dropdown-item" href="#" appStopClick (click)="copy(c, c.login.totp, 'verificationCodeTotp', 'TOTP')"
+                                    *ngIf="c.login.hasTotp">
```

Comment:
Need to condition based on capabilities, etc.

see: https://github.com/bitwarden/jslib/blob/master/src/angular/components/view.component.ts#L92

Reply:
I'm not sure I understand, doesn't the c.login.hasTotp condition sufficiently? Are you looking for an explicity `cipher.type` check?